src/lib/types.ts

```
@@ -1,7 +1,12 @@
 export interface Port {
 	portName: string;
 	speedGbps?: number; // default to 1 if not provided
-	connectedTo?: string; // e.g. "Router-LAN2"
+	connectedTo?: ConnectedPortRef;
```

ESLint

[error] 'ConnectedPortRef' was used before it was defined.

(no-use-before-define)